lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -485,6 +497,18 @@ instance FromJSON (ApiT WalletState) where
 instance ToJSON (ApiT WalletState) where
     toJSON = genericToJSON walletStateOptions . getApiT
 
+instance FromJSON ApiUtxoStatistics where
+    parseJSON = genericParseJSON defaultRecordTypeOptions
+instance ToJSON ApiUtxoStatistics where
+    toJSON = genericToJSON defaultRecordTypeOptions
+
+instance ToJSON (ApiT BoundType) where
+    toJSON (ApiT Log10) = "log10"
+
+instance FromJSON (ApiT BoundType) where
+    parseJSON "log10" = pure $ ApiT Log10
+    parseJSON _ = fail "wrong BoundType value"
+
```

Comment:
Note that both suggestions above will require a small addition to `defaultSumTypeOptions`: 

```hs
 defaultSumTypeOptions :: Aeson.Options
 defaultSumTypeOptions = Aeson.defaultOptions
     { constructorTagModifier = camelTo2 '_'
+    , tagSingleConstructors = True
     }
```

This should have no consequences on other parsers.